### ISSUE TITLE
Will you open source code for IThreadPool?

### ISSUE THREAD

**Issue by fowuyu** [fabri-guo/base#1]
Will you open source code for IThreadPool?

**Comment by fabri-guo** [comment]
No, I'm just starting my work now, and the goal is just to make a universal method library that can be easily used in later development




------------------ 原始邮件 ------------------
发件人: "易辰"<notifications@github.com>; 
发送时间: 2019年9月30日(星期一) 下午5:37
收件人: "fabri-guo/base"<base@noreply.github.com>; 
抄送: "Subscribed"<subscribed@noreply.github.com>; 
主题: [fabri-guo/base] Will you open source code for IThreadPool? (#1)




Will you open source code for IThreadPool?
 
&mdash;

**Comment by fabri-guo** [comment]
I will open source the code that will be developed later, just to understand that there are some problems with what you said




------------------ 原始邮件 ------------------
发件人: "易辰"<notifications@github.com>;
发送时间: 2019年9月30日(星期一) 下午5:37
收件人: "fabri-guo/base"<base@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>;
主题: [fabri-guo/base] Will you open source code for IThreadPool? (#1)




Will you open source code for IThreadPool?
